apps/source_academy/priv/repo/migrations/20170626081409_create_users.exs

```diff
@@ -0,0 +1,15 @@
+defmodule SourceAcademy.Repo.Migrations.CreateUsers do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :name, :string
+      add :email, :string
+      add :is_admin, :boolean, default: false
```

Comment:
Validate role in changeset? I thought the flow is such that it is either empty (defaults to "student" in db level) or changed into staff via `make_staff` call? Or do we want a `select` field in the registration form?

Also, if you want `role` validation in db level, we need [another dep](https://github.com/gjaldon/ecto_enum) because ecto [doesn't support it out-of-the-box](https://elixirforum.com/t/how-to-have-predefined-values-select-in-an-ecto-field/2086)